pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -3013,6 +3013,19 @@ pj_status_t pjsua_media_channel_create_sdp(pjsua_call_id call_id,
         sdp->bandw[sdp->bandw_count++] = b;
     }
 
+#if defined(PJMEDIA_HAS_RTCP_XR) && (PJMEDIA_HAS_RTCP_XR != 0)
+    if (acc->cfg.enable_rtcp_xr) {
+        pjmedia_sdp_attr *a;
+        const char *STR_RTCP_XR = "rtcp-xr";
+        const pj_str_t value =
+            pj_str("pkt-loss-rle pkt-dup-rle pkt-rcpt-times rcvr-rtt "
+                   "stat-summary voip-metrics");
```

Comment:
AFAIK currently we only support the last 3 block types (rtt, stat summary & voip metrics).